src/stylesheets/screens/home-details.scss

```diff
@@ -0,0 +1,23 @@
+.home-details {
+  @include details-container();
+  position: relative;
+
+  flex: 1;
```

Comment:
minor thing but do we have any reason for line breaking here? is it intended?

Reply:
@namtx  as you guessed, It is intended. It was implemented in - https://github.com/rafayet-monon/surveyor/pull/13
all styles are now ordered in a specific way. We used this stylelint plugin to implement it - https://github.com/constverum/stylelint-config-rational-order